src/styles/utils/css-reset.scss

```diff
@@ -0,0 +1,23 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
you should reset indentation directly for that selector, that you need instead of *